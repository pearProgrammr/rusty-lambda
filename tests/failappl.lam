a := (\ x. x);
b := 1;
b a;
