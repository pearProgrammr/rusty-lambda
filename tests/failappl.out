Error: Custom { kind: InvalidInput, error: StringError("terms need to be applied to function types") }
